tests/PatternAssertions/FileTeaserAssert.php

```diff
@@ -29,23 +30,6 @@ protected function getAssertions($variant): array {
     return $assertions;
   }
 
-  /**
-   * Asserts the image of the pattern.
-   *
-   * @param array|null $expected_image
-   *   The expected image.
-   * @param \Symfony\Component\DomCrawler\Crawler $crawler
-   *   The DomCrawler where to check the element.
-   */
-  protected function assertImage($expected_image, Crawler $crawler): void {
-    if (is_null($expected_image)) {
-      $this->assertElementNotExists('div.ecl-file--thumbnail div.ecl-file__container div.ecl-file__detail div.ecl-file__image img', $crawler);
```

Comment:
There is a problem here, this assertion was wrong (notice the difference between the css selector and the next) and now that it was fixed (by moving it into the generic method) the tests on ewcms will fail. We need a PR on ewcms to fix all the tests where this assert class is used.

Reply:
EWCMS PR https://github.com/ec-europa/ewcms/pull/903